docs/docs/how-to/self-hosting-redwood.md

```diff
@@ -1,3 +1,5 @@
+> NOTE: This doc has been deprecated in favor of the [Baremetal](/docs/docs/deploy/baremetal.md) docs
```

Comment:
@chapmanm3 thanks! I think we should use an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) instead of a blockquote, and here's how you make the link relative to fix the Netlify build:

```suggestion
:::caution This doc has been deprecated

This doc has been deprecated in favor of the [Baremetal](../deploy/baremetal.md) docs.

:::
```

Reply:
Perfect thanks so much!